bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,382 @@
+
+# Pages templates
+# Game embed layout
+GAME_PAGE = Template(
+    textwrap.dedent("""
+        **[${name}](${url})**
+        ${description}
+        **Release Date:** ${release_date}
+        **Rating:** ${rating}/100 :star: (based on ${rating_count} ratings)
+        **Platforms:** ${platforms}
+        **Status:** ${status}
+        **Age Ratings:** ${age_ratings}
+        **Made by:** ${made_by}
+
+        ${storyline}
+    """)
+)
+
+# .games company command page layout
+COMPANY_PAGE = Template(
+    textwrap.dedent("""
+        **[${name}](${url})**
+        ${description}
+        **Founded:** ${founded}
+        **Developed:** ${developed}
+        **Published:** ${published}
+    """)
+)
+
+# For .games search command line layout
+GAME_SEARCH_LINE = Template(
+    textwrap.dedent("""
+        **[${name}](${url})**
+        ${rating}/100 :star: (based on ${rating_count} ratings)
+    """)
+)
+
+# URL templates
+COVER_URL = Template("https://images.igdb.com/igdb/image/upload/t_cover_big/${image_id}.jpg")
+LOGO_URL = Template("https://images.igdb.com/igdb/image/upload/t_logo_med/${image_id}.png")
+
+# Create aliases for complex genre names
+ALIASES = {
+    "Role-playing (rpg)": ["Role-playing", "Rpg"],
+    "Turn-based strategy (tbs)": ["Turn-based-strategy", "Tbs"],
+    "Real time strategy (rts)": ["Real-time-strategy", "Rts"],
+    "Hack and slash/beat 'em up": ["Hack-and-slash"]
+}
+
+
+class GameStatus(IntEnum):
+    """Game statuses in IGDB API."""
+
+    Released = 0
+    Alpha = 2
+    Beta = 3
+    Early = 4
+    Offline = 5
+    Cancelled = 6
+    Rumored = 7
+
+
+class AgeRatingCategories(IntEnum):
+    """IGDB API Age Rating categories IDs."""
+
+    ESRB = 1
+    PEGI = 2
+
+
+class AgeRatings(IntEnum):
+    """PEGI/ESRB ratings IGDB API IDs."""
+
+    Three = 1
+    Seven = 2
+    Twelve = 3
+    Sixteen = 4
+    Eighteen = 5
+    RP = 6
+    EC = 7
+    E = 8
+    E10 = 9
+    T = 10
+    M = 11
+    AO = 12
+
+
+class Games(Cog):
+    """Games Cog contains commands that collect data from IGDB."""
+
+    def __init__(self, bot: SeasonalBot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+        # Initialize genres
+        bot.loop.create_task(self._get_genres())
+
+    async def _get_genres(self) -> None:
+        """Create genres variable for games command."""
+        body = "fields name; limit 100;"
+        async with self.http_session.get(f"{BASE_URL}/genres", data=body, headers=HEADERS) as resp:
+            result = await resp.json()
+
+        genres = {genre["name"].capitalize(): genre["id"] for genre in result}
+
+        self.genres = {}
+
+        # Replace complex names with names from ALIASES
+        for genre in genres:
+            if genre in ALIASES:
+                for alias in ALIASES[genre]:
+                    self.genres[alias] = genres[genre]
+            else:
+                self.genres[genre] = genres[genre]
```

Comment:
Since we only do `_get_genres()` **once**, if this fails we have to reload the entire cog everytime, so I suggest turning `self.genres` into a `property` instead like so
```py
@property
def gernes(self):
    ...
```
This way, we can also add another command to refresh `genres` whenever we want to.

Or, you can create a `discord.ext.tasks` that loop every x amount that will fetch the genres, and of course, a command to refresh `self.genres`.